packages/twenty-server/src/workspace/messaging/repositories/workspace-member/workspace-member.service.ts

```diff
@@ -47,4 +49,26 @@ export class WorkspaceMemberService {
 
     return workspaceMembers[0];
   }
+
+  public async getAllByWorkspaceIdOrFail(
```

Comment:
nit: Not sure if you want to fail here if the collection is empty? Usually it's good to have this when you want to reach a specific id (like the first entry with getFirstOrFail) because you don't want to have undefined however for collections you'll most likely loop over / use maps which should be null safe

Reply:
WorkspaceId is kind of implicit here